### PR TITLE
Expose more NUnitConsole arguments in NUnit3 for testing and tuning

### PR DESCRIPTION
**Pull request by jasonleenaylor**
* Also add instructions for packing the project for local testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.3.5] - 2021-01-13
```
Comment (ermshiperete) [inline CHANGELOG.md]:
```suggestion
## [2.3.5] - 2021-01-22
```

File: Documentation/SIL.BuildTasks.md [hunk 2]
```diff
@@ -161,13 +161,27 @@ Runs NUnit3 on a test assembly.
...
+- `UseNUnit3Xml`: Whether to use the NUnit3 or NUnit2 XML format
+
 - `TeamCity`: Should be set to true if the tests are running on a TeamCity server.
   Adds `--teamcity` when calling nunit which _"Turns on use of TeamCity service messages."_
 
+- `Agent`: The number of NUnit agents to use when running the tests
+
+- `Workers`: Specify the NUMBER of worker threads to be used in running tests
+
+- `Process`: PROCESS isolation for test assemblies. Values: Single, Separate, Multipl
```
Comment (ermshiperete) [inline Documentation/SIL.BuildTasks.md]:
```suggestion
- `Process`: PROCESS isolation for test assemblies. Values: Single, Separate, Multiple
```

File: Readme.md [hunk 3]
```diff
@@ -44,3 +44,9 @@ Run tests:
 ```bash
 msbuild build/SIL.BuildTasks.proj
 ```
+
+### Building a local package of SIL.BuildTasks for testing (Windows)
+
+1. Run the Pack build command from inside Visual Studio on the SIL.ReleaseTasks.Dogfood project
+2. Run the Pack build command from inside Visual Studio on the SIL.BuildTasks project
+3. Install the package from the output folder into your local Nuget source e.g.`nuget.exe add output/Debug/SIL.BuildTasks.2.3.5-beta.1.nupkg -Source /c/Repositories/DevelopmentPackages/`
```
Comment (ermshiperete) [inline Readme.md:52]: Simply copying the package into the nuget source folder also works

File: CHANGELOG.md (hunk 1 above)
Comment (ermshiperete) [inline CHANGELOG.md]: Technically this adds new functionality, so per semver this should increase the minor version number.

```suggestion
## [2.4.0] - 2021-01-22
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -16,6 +16,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
+
+### Changed
```
Comment (ermshiperete) [inline CHANGELOG.md]:
```suggestion
### Added
```

**Review by ermshiperete — COMMENTED** [review]
A few suggestions, but overall :lgtm:

File: CHANGELOG.md [hunk 5]
```diff
@@ -16,6 +16,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.3.5] - 2021-01-22
```
Comment (ermshiperete) [inline CHANGELOG.md]:
```suggestion
## [2.4.0] - 2021-01-22
```

File: Readme.md (hunk 3 above)
Comment (jasonleenaylor) [reply Readme.md:52]: I feel like using the add command encourages understanding/investigation of the local source feature

**Pull request merged**